hoptimator-catalog/src/main/java/com/linkedin/hoptimator/catalog/AvroConverter.java

```diff
@@ -21,7 +21,7 @@ public final class AvroConverter {
   private AvroConverter() {
   }
 
-  public static Schema avro(String name, String namespace, RelDataType dataType) {
+  public static Schema avro(String namespace, String name, RelDataType dataType) {
```

Comment:
I believe that L#28 (inside this method) is reversing the `namespace` and `name` parameters.

Reply:
nice catch. Figured I'd miss one somewhere.